iotdb-core/datanode/src/main/java/org/apache/iotdb/db/protocol/thrift/impl/DataNodeInternalRPCServiceImpl.java

```diff
@@ -2157,7 +2157,9 @@ private TSStatus createNewRegionPeer(ConsensusGroupId regionId, List<Peer> peers
       if (regionId instanceof DataRegionId) {
         DataRegionConsensusImpl.getInstance().createLocalPeer(regionId, peers);
       } else {
-        SchemaRegionConsensusImpl.getInstance().createLocalPeer(regionId, peers);
+        final List<Peer> createParam =
```

Comment:
I recommend not adding the isRatis function here, but instead just checking the consensus algorithm to see if an empty list should be passed in

Reply:
done